From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Mon, 9 Sep 2019 01:29:27 +0900
Subject: udev: also logs file permission

(cherry picked from commit 20eef7c55cfdd83f953c6a332aaad0caba4fb60f)
(cherry picked from commit 4b92ffccaa865382f18d0e98cecec1ab1085759e)
---
 src/udev/udev-rules.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/udev/udev-rules.c b/src/udev/udev-rules.c
index 21f84b2..8631a26 100644
--- a/src/udev/udev-rules.c
+++ b/src/udev/udev-rules.c
@@ -2304,7 +2304,7 @@ static int apply_static_dev_perms(const char *devnode, uid_t uid, gid_t gid, mod
                 return log_error_errno(errno, "Failed to chown '%s' %u %u: %m",
                                                device_node, uid, gid);
         else
-                log_debug("chown '%s' %u:%u", device_node, uid, gid);
+                log_debug("chown '%s' %u:%u with mode %#o", device_node, uid, gid, mode);
 
         (void) utimensat(AT_FDCWD, device_node, NULL, 0);
         return 0;
